package.json

```diff
@@ -16,6 +16,12 @@
     "lint-staged": "^12.4.1"
   },
   "dependencies": {
+    "@fortawesome/fontawesome-free": "^6.1.1",
```

Comment:
If these packages aren't being used anymore, they should be removed

Reply:
for some reason, i don't see the packages on my local version anymore, and it says that the branch is up to date...

Reply:
Did you make sure you added the files to tracking? go into root directory and `git add *` and commit

Reply:
i am so sorry, i was looking at the .json file in the client directory, i will remove these right away!